dev/reference/sql/statements/create-database.md

```diff
@@ -76,4 +76,4 @@ mysql> SHOW TABLES;
 * [USE](/reference/sql/statements/use.md)
 * [ALTER DATABASE](/reference/sql/statements/alter-database.md)
 * [DROP DATABASE](/reference/sql/statements/drop-database.md)
-* [SHOW CREATE DATABASE](/reference/sql/statements/show-create-database.md)
+* [SHOW DATABASES](/reference/sql/statements/show-databases.md)
```

Comment:
Please also help update this file in `v2.1`.

Reply:
Added this link to v2.1 file, PTAL again.